src/common/async-job-runner.js

```diff
@@ -96,6 +96,11 @@ export class AsyncJobRunner extends StepAudit {
       };
 
       updatedStepContext.finalUrl = await this.urlResolver(site, context);
+      log.info(`site: ${siteId}. Final URL: ${updatedStepContext.finalUrl} and delivery type: ${site.getDeliveryType()} and promiseToken: ${message.promiseToken}`);
```

Comment:
@anagarwa , Please make sure to remove `promiseToken` from the logs after debugging.